src/imitation/scripts/common/demonstrations.py

```diff
@@ -62,11 +61,14 @@ def generate_expert_trajs(
     if n_expert_demos is None:
         raise ValueError("n_expert_demos must be specified when rollout_path is None")
 
-    rollout_env = DummyVecEnv(
-        [lambda: RolloutInfoWrapper(gym.make(common["env_name"])) for _ in range(4)],
+    rollout_env = vec_env.DummyVecEnv(
```

Comment:
I thought I commented about this before but can't see it now -- don't we need to seed the environments individually, to make it (a) replicable, (b) uncorrelated between environments?

Would suggest using an existing helper method from `util` for this rather than rolling our own.

Reply:
Yes you commented before. I fixed it  by using `common.make_venv()` instead.